### ISSUE TITLE
[feature] swipe delete

### ISSUE THREAD

**Issue by Quantenzitrone** [Julow/Unexpected-Keyboard#46]
https://user-images.githubusercontent.com/74491719/151580765-c79ab179-e3c0-4618-9c0c-6414d38b7ed4.mp4

like in this video of OpenBoard.

**Comment by Julow** [comment]
This would be an awesome feature. (similar suggestion: https://github.com/Julow/Unexpected-Keyboard/issues/7)

However, I don't want to copy this idea as it is because it wouldn't be the best fit for this keyboard. There's already a "precise repeat" feature that is not working well and I planned to improve it at some point.

**Comment by Julow** [comment]
The "precise repeat" feature will be improved during the next release but maybe not enough to close this issue yet. (c85e9a9)

**Comment by Quantenzitrone** [comment]
i think this issue is resolved enough with "precise repeat" and Ctrl+Backspace (delete word)

**Comment by Julow** [comment]
You might be interested in https://github.com/Julow/Unexpected-Keyboard/issues/7#issuecomment-1399625673

**Comment by moya2162** [comment]
I would like to add my opinion that this feature would be nice to have as well.

Swipe to move cursor which was recently added is a huge welcome. Shift + swipe to highlight a word is nice as well when trying to specifically delete a word somewhere in a sentence. 

I would like to add that for speed and ergonomics, it would be nice to have swipe on delete to quickly delete a mistake. Its faster than having to hit shift, highlight, hit delete which is 3 gestures vs 1. My humble 2 cents.

**Comment by Julow** [comment]
This seems nice indeed.

**Comment by mexicancartel** [comment]
Came here to request this feature. I keeps switching back to simple keyboard for regular use because of this.

**Comment by Sx1Fl0NdqlkM6N4s2pswXKUI0k9FKTaOckLbNXu** [comment]
I don't know if keys like "select all" are somehow fundamentally different than "key = key combination", but having access to those would mostly solve the issue:

ctrl+backspace, ctrl+delete = delete word
ctrl+left, ctrl+right = move by word
ctrl+shift+left, ctrl+shift+right = select word
shift+left, shift+right = select character

unless we can already send key combinations?

**Comment by ChristianKurz** [comment]
I'm also missing this feature from simple keyboard. I don't  think key combinations are a good replacement - swipe deletion with selecfion is fast and very precise.

**Comment by evantj** [comment]
setting w="selection_cursor_left" on the backspace key and then just typing over the selection works pretty well for me

**Comment by FinderN** [comment]
> setting w="selection_cursor_left" on the backspace key and then just typing over the selection works pretty well for me

Yes, it is nice when there is `w=" :selection_cursor_left (on key up) backspace"`

**Comment by toboil-features** [comment]
Any progress or fork, possibly build of keyboard with this feature? Feels really usable when working with text.
This feature and existing functionality of "unexpected" is a killer combo for me.
